icaruscode/TPC/ICARUSWireCell/detsimmodules_wirecell_ICARUS.fcl

```diff
@@ -85,4 +85,8 @@ icarus_simwire_wirecell_fitSR.wcls_main.structs.gain0: 11.9918701 # mV/fC
 icarus_simwire_wirecell_fitSR.wcls_main.structs.gain1: 12.6181926 # mV/fC
 icarus_simwire_wirecell_fitSR.wcls_main.structs.gain2: 13.0261362 # mV/fC
 icarus_simwire_wirecell_fitSR.wcls_main.structs.shaping1: 1.45 # us
+
+icarus_simwire_wirecell_filtersed: @local::icarus_simwire_wirecell
+icarus_simwire_wirecell_filtersed.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel-refactored-filtersed.jsonnet"] 
+
```

Comment:
Shouldn't this be in a new fcl or shouldn't the detsim fcl also be modified to run the filter module?